pkg/cloud/vsphere/services/govmomi/vcenter/clone.go

```diff
@@ -187,16 +186,16 @@ func getNetworkSpecs(
 		// "types.BaseVirtualEthernetCard" as a "types.BaseVirtualDevice".
 		nic := dev.(types.BaseVirtualEthernetCard).GetVirtualEthernetCard()
 
+		if netSpec.MACAddr != "" {
+			nic.MacAddress = netSpec.MACAddr
+			nic.AddressType = "Manual"
```

Comment:
Is this a recognized address type by vSphere or arbitrary?

Reply:
Hi @andrewsykim,

It's one of [three, valid values](https://www.vmware.com/support/developer/converter-sdk/conv60_apireference/vim.vm.device.VirtualEthernetCard.html#addressType).

Reply:
Hi @andrewsykim,

Please check now to see if this helps.

Reply:
Note that there is a generated constant you can use: `types.VirtualEthernetCardMacTypeManual`

Reply:
^ a good TODO for @james65535